### PR TITLE
Add the Red, White, Blue Classic Color Scheme for Classic Theme

### PR DESCRIPTION
**Pull request by milawynsrealm**
## Purpose

The purpose of this commit is to add the Red, White, and Blue Color Scheme that's currently missing from the list. Since this color scheme comes with Windows XP, it would make sense to add it since someone may expect it to be there. In addition, I also translated the strings for all the Color Schemes into themeui.

JIRA issue: [CORE-12943](https://jira.reactos.org/browse/CORE-12943)

## Proposed changes

- Adds the 'Red, White, and Blue' Color Scheme to the registry.
- Adds translations of Color Schemes for the Classic Theme into themeui.dll.

### CODE REVIEW

**Comment by HBelusca** [comment]
Out of curiosity, where do the translations come from?

**Comment by milawynsrealm** [comment]
The translations come from the hivedef.inf file itself. From the translations DESKTOP_SCHEME_X (replace X with a number). I just hope I didn't do any copy paste mistakes.

File: boot/bootdata/hivedef.inf [hunk 1]
```diff
@@ -289,7 +289,7 @@ HKCU,"Control Panel\Appearance\New Schemes\0\Sizes\0","Size #6",0x000b0001,0c,00
 HKCU,"Control Panel\Appearance\New Schemes\0\Sizes\0","Size #7",0x000b0001,12,00,00,00,00,00,00,00
 HKCU,"Control Panel\Appearance\New Schemes\0\Sizes\0","Size #8",0x000b0001,12,00,00,00,00,00,00,00
 ; Brick
-HKCU,"Control Panel\Appearance\New Schemes\1","DisplayName",0x00020000,"@themeui.dll,-851"
+HKCU,"Control Panel\Appearance\New Schemes\1","DisplayName",0x00020000,"@themeui.dll,-850"
```
Comment (HBelusca) [inline boot/bootdata/hivedef.inf]: Were the old values (851 in this case) compatible with Windows Server 2003 ones?

Comment (milawynsrealm) [reply boot/bootdata/hivedef.inf]: No, in Windows XP the value was
> @themeui.dll,-2019
But at the time, I didn't think it mattered if the value was exact or not. Want me to change it? Just keep in mind that this month is extremely busy for me.

Comment (HBelusca) [reply boot/bootdata/hivedef.inf]: No problem, don't change anything. I was just wondering whether the original values were from somewhere, or whether they were just chosen arbitrarily.

**Pull request merged**